vmdb/app/helpers/application_helper.rb

```diff
@@ -1003,7 +1003,7 @@ def build_toolbar_hide_button(id)
       when "vm_guest_restart", "instance_guest_restart"
         return true if !@record.is_available?(:reboot_guest)
       when "vm_migrate", "vm_reconfigure"
-        return true if @record.vendor.downcase == "redhat"
+        return true if %w(microsoft redhat).include?(@record.vendor.downcase)
```

Comment:
Same here...if the model has a `supports_migrate?` method (and/or `supports_reconfigure?`), then this becomes `return true if @record.supports_migrate? || @record.supports_reconfigure?`